sql-alwayson-md-ilb-zones/.ci_skip

```diff
@@ -0,0 +1,2 @@
+Skip CI due to existing resource requirements
```

Comment:
See the readme in the repo for how to handle existing resources/prereqs...  I'd like to see if we can get this validated in CI...

Reply:
This approach will be challenging for this template, because an existing Active Directory domain environment is also required - the prereq's folder would need to include the equivalent of an entire quickstart template for deploying Windows Server AD domain controllers.  We've already successfully tested this SQL template with several deployments in multiple regions, so the template itself is solid. Can we approve as-is?

Reply:
Let's provide some instructions in the readme on how to get a successful deployment...

Reply:
I've updated the README with the instructions needed to deploy the prerequisites.  Is this sufficient?

Reply:
Let's add the prereq templates - the az zones templates are simple enough that it should work fine (unless the deployment takes too long) - we're going to start automating deployment and reporting status and ci_skip will make it show as "not tested"

Reply:
Would it be acceptable to have the prereq template call the azuredeploy.json for the existing AD-zones Quick Start template, rather than copying the entire AD-zones template into the prereq folder of this template?

Reply:
@bmoore-msft Could you comment on the approach of linking to the existing AD zones template in the Quick Start gallery as a prereq vs copying the entire AD zones template into the prereq folder of this template?  Is linking an acceptable approach?  We just trying to avoid create multiple copies of what is essentially the same template code that then increases maintenance over time.

Reply:
The engine doesn't support that right now, and I'm not sure we will since a change in one of those templates could break dependencies... since the prereqs are static, copying should be fine.